tests/components/zwave_js/test_binary_sensor.py

```diff
@@ -159,7 +151,7 @@ async def test_property_sensor_door_status(hass, lock_august_pro, integration):
     state = hass.states.get(PROPERTY_DOOR_STATUS_BINARY_SENSOR)
     assert state
     assert state.state == STATE_OFF
-    assert state.attributes["device_class"] == DEVICE_CLASS_DOOR
+    assert state.attributes["device_class"] == BinarySensorDeviceClass.DOOR
```

Comment:
I see that some tests use "device_class" and others use `ATTR_DEVICE_CLASS` .
Could you standardise with `ATTR_DEVICE_CLASS`?

Reply:
Agreed, done.